Corona-Warn-App/src/main/res/layout/fragment_onboarding_tracing.xml

```diff
@@ -2,6 +2,12 @@
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
+    <data>
+        <variable
+            name="interopViewModel"
```

Comment:
I would write it out, as just `interop` could be confusing for a new developer -> `interoperabilityViewModel`